pkg/csi/types/envvars.go

```diff
@@ -22,6 +22,9 @@ const (
 )
 
 const (
+	// EnvCSIEndpoint points to the location of the unix socket
+	EnvCSIEndpoint = "CSI_ENDPOINT"
+
```

Comment:
No need to define this here at all, it is already available through `gocsi`

Reply:
Will use the gocsi label